### ISSUE TITLE
Arch package fails to launch

### ISSUE THREAD

**Issue by Technetium1** [anthraxx/cowfortune#4]
[cowfortune](https://archlinux.org/packages/extra/any/cowfortune/) `0.1.2-8` fails to launch due to the following error:
```
ls: cannot access '/usr/share/cows': No such file or directory
cowsay: Could not find cowfile for ''!
```
Might need a new tag: https://github.com/anthraxx/cowfortune/compare/0.1.2...master

**Comment by Christoffer-JL** [comment]
I am getting this too!

**Comment by hexive** [comment]
anyone know a quick fix for this?

**Comment by Amidamaroo** [comment]
Yes, you need to copy the cows so that you have cows on the path /usr/share/cows and /usr/share/cowsay/cows. Works for me

**Comment by DanielChesters** [comment]
Hi, you can start the program on Arch with COWPATH env variable:

`COWPATH=/usr/share/cowsay/cows/ cowfortune`
